Update tenant-rename.md (#2809)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/tenant-rename.md

It's now possible to change the SharePoint domain name for your organization in Microsoft 365. For example, if the name of your organization changes from "Contoso" to "Fabrikam," you can change `contoso.sharepoint.com` to `fabrikam.sharepoint.com`.

> [!IMPORTANT]
> This feature is only available for customers that have been accepted into the Private Preview program. If you would like to nominate yourself into the Private Preview you can do so at https://aka.ms/SPOTenantRenameNomination. Please note that the nomination does not necessarily mean that you will be accepted. We are reviewing these on a case-by-case basis and will only be in contact if you are accepted.

> [!IMPORTANT]

> > This feature is available isn't available for organizations that have set up Multi-geo.

Reviewer: ```suggestion
> This feature is not available for organizations that have set up a multi-geo.
```

> 
> > [!IMPORTANT]
> > This feature is available isn't available for organizations that have set up Multi-geo.

Reviewer: ```suggestion
> - This feature is not available for organizations that have set up a multi-geo.
```
